### DISCUSSION TITLE
TODO list for the next version

### DISCUSSION THREAD

**Discussion by FilipSLO** [micropython/micropython#12340] [Core Development] [answered]
Hi, I search unsuccessfully if there is such a list. 

So my question is: what's the release policy or plan?

**Comment by andrewleech** [comment, marked as answer]
They're being tracked in milestones, see https://github.com/micropython/micropython/issues?q=milestone%3Arelease-1.21.0

**Reply by Josverl** [reply]
Pretty sure there's more than these fixes in 1.20.1, but glad there is some visibility.

I'd love projects to be used to understand were I could help, and what to is postponed to a next release ..
But there I do understand that there is effort, and some Grey noise associated with that as well

**Reply by andrewleech** [reply]
Oh yes there's always a lot more included / merged into a release, I believe the milestones are being used to track "highlight" or blocking features that are treated as required before a release can be cut.

I'd never heard of [Github Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) I'm not sure why that was done as a standalone feature separate to milestones, but sure it's _another_ way to try to plan and track tasks.  I suppose it's got a lot more visibility than milestones (it took me a while to find the milestone and I knew what I was looking for) so it might be worth looking at if a basic usage can be as minimal overhead as milestones.

The reality is, yeah, this level of project management very quickly becomes a full time job... and personally I want the current core micropython team to still have time to do coding!
